src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -165,8 +165,6 @@ def get(self, request: Request, organization: Organization | RpcOrganization) ->
 
         # ========================================================================
         # Step 2: Fetch releases, prefetch projects & release_thresholds
-        # NOTE: we're only filtering on date ADDED
-        # This is not synonymous with a deploy... which may be what we actually want.
         # ========================================================================
         release_query = Q(organization=organization, date_added__gte=start, date_added__lte=end)
         if environments_list:
```

Comment:
We're querying for 
```
Q(
                releaseprojectenvironment__environment__name__in=environments_list,
            )
```

🤔 Do i need to prefetch for `releaseprojectenvironment_set` then?

Reply:
is this to improve performance? i don't think it could hurt